### ISSUE TITLE
NTR: splenopancreatic fusion

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4156]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Splenopancreatic fusion

### 2. Synonyms
Spleno-pancreatic fusion
Splenic pancreatic fusion

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Fusion of the pancreatic tail and spleen.
PMID:18398855 

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0025408 'Abnormal spleen morphology'
HP_0012090 'Abnormality of pancreas morphology'

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Schinzel-Giedion syndrome (PMID:18398855)
trisomy 13 (PMID:28843273)

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
https://orcid.org/0000-0001-5208-3432

This is for the Kid's First Data Portal - the description is 'Splenopancreatic fusion (microscopic)'
